src/docling_extractor_v2.py

```
@@ -371,6 +377,269 @@ def _setup_converter(self) -> DocumentConverter:
+                    width,
+                    height,
+                    text_value.strip(),
+                )
+
+            if angle < min_angle or angle > max_angle:
+                continue
+
+            try:
+                text_item.text = ""
+            except Exception:
+                continue
```

Ruff (0.14.3)

`try`-`except`-`continue` detected, consider logging the exception

(S112)